src/chrome/chromeDebugAdapter.ts

```diff
@@ -449,32 +453,39 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
      * This event tells the client to begin sending setBP requests, etc. Some consumers need to override this
      * to send it at a later time of their choosing.
      */
-    protected sendInitializedEvent(): void {
+    protected async sendInitializedEvent(): Promise<void> {
         // Wait to finish loading sourcemaps from the initial scriptParsed events
         if (this._initialSourceMapsP) {
             const initialSourceMapsP = this._initialSourceMapsP;
             this._initialSourceMapsP = null;
 
-            initialSourceMapsP.then(() => {
+            await initialSourceMapsP.then(async () => {
```

Comment:
The mixing of async/await and promises is a little confusing here (`await ... .then(`)

Reply:
Done